tools/postman-collections/am-lib-testing.postman_collection.json

```diff
@@ -130,6 +130,34 @@
 				}
 			},
 			"response": []
+		},
+		{
+			"name": "Revoke Resource Access",
```

Comment:
why do we use tabs not whitespaces?

Reply:
just how postman has exported. Agreed tabs are unecessary. Reformatted